components/Navbar.jsx

```diff
@@ -59,6 +32,15 @@ const Navbar = () => {
           {/* <Link href="/partnerships">
             <span className={TEXT_CLASSES}>Partnerships</span>
           </Link> */}
+          <Link href="/who-we-are">
+            <span className={TEXT_CLASSES}>About</span>
+          </Link>
+          <Link href="/events">
+            <span className={TEXT_CLASSES}>Events</span>
+          </Link>
+          <Link href="/projects">
+            <span className={TEXT_CLASSES}>Project</span>
+          </Link>
```

Comment:
I think you'll also have to add these links in `MobileNavbarDrawer.jsx` and remove the extra dropdown (I did not make it like that, there just so happens to be two nabbers)

Reply:
There's some dead code in that file too you could probably get rid of as well